uber/reset_db.py

```diff
@@ -1,26 +0,0 @@
-from uber.common import *
```

Comment:
I moved all of the ``entry_point``s into a single module.  People kept creating a new file every time they needed to add one, and my personal preference is to group such things together (at least until a file gets too long, which for me is at least a thousand lines of code, thought tastes differ).